.github/workflows/publish-specs.yml

```diff
@@ -79,4 +78,5 @@ jobs:
           export PATH="$(python3 -m site --user-base)/bin:${PATH}"
           bikeshed update
           mkdir out
-          ./compile.sh
+          cd wgsl
```

Comment:
nit: `make -C wgsl` to avoid a cd? (state change in the middle of a script can sometimes be confusing)

Reply:
Right, reads way better. Thanks!